tangelo/toolboxes/molecular_computation/integral_solver_pyscf.py

```diff
@@ -108,14 +111,21 @@ def compute_mean_field(self, sqmol):
         sqmol.mean_field = self.scf.RHF(molecule) if not sqmol.uhf else self.scf.UHF(molecule)
         sqmol.mean_field.verbose = 0
 
+        if self.chkfile is not None:
```

Comment:
`is_chkfile_exist` -> `chkfile_found`

Suggestion: define variable with default value out of branch. Update if necessary.

```
chkfile_found = False
if self.chkfile:
   chkfile_found = os.path.exists(self.chkfile)
   sqmol.mean_field.chkfile = self.chkfile
```